run.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 
-mkdir -p ${DOWNLOAD_DIR}
-gsutil cp ${SOURCE_PATH} ${DOWNLOAD_DIR}
+if [ ! -z "${DOWNLOAD_DIR}" ] && [ ! -d "${DOWNLOAD_DIR}" ]; then
+  mkdir -p "${DOWNLOAD_DIR}"
+fi
+if [ ! -z "${SOURCE_PATH}" ] && [ -d "${DOWNLOAD_DIR}" ]; then
+  gsutil cp "${SOURCE_PATH}" "${DOWNLOAD_DIR}" && exec crond -f -c /root/crontabs
+fi
+gsutil "$@" && exec crond -f -c /root/crontabs
```

Comment:
Ah, as crond never stops (because of `-f` option) I thought `else` clause is not necessary. But once crond failed (or in case signaled?), the line may be executed afterwards! I'm going to fix it. Thank you!